### PR TITLE
update deploy and example instructions

### PR DESCRIPTION
**Pull request by edisonxiang**
update deploy and example instructions for e2e test.

### CODE REVIEW

**Comment by edisonxiang** [comment]
@rootfs @chakri-nelluri, welcome to review , your suggestions are very important. Thanks.

**Comment by dims** [comment]
@chakri-nelluri looks like this is ready to merge

File: pkg/cinder/README.md [hunk 1]
```diff
@@ -1,19 +1,58 @@
...
+
+Mountprogpation requries support for privileged containers. So, make sure privileged containers are enabled in the cluster.
+
+### Example local-up-cluster.sh
+
+```ALLOW_PRIVILEGED=true FEATURE_GATES=CSIPersistentVolume=true,MountPropagation=true RUNTIME_CONFIG="storage.k8s.io/v1alpha1=true" LOG_LEVEL=5 hack/local-up-cluster.sh```
+
+### Deploy
+
+Please update ```cloud.conf``` informations in csi-configmap-cinderplugin.yaml file.
```
Comment (chakri-nelluri) [inline pkg/cinder/README.md]: replace "informations" with "configuration"

File: pkg/cinder/README.md [hunk 2]
```diff
@@ -1,19 +1,58 @@
...
+
+```kubectl -f deploy/kubernetes/csi-configmap-cinderplugin.yaml create```
```
Comment (chakri-nelluri) [inline pkg/cinder/README.md]: can't we do ```kubectl -f deploy/kubernetes``` ?

File: pkg/cinder/deploy/kubernetes/csi-configmap-cinderplugin.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# This YAML file contains configmap objects,
+# which are necessary to run csi cinder plugin.
+
+kind: ConfigMap
```
Comment (chakri-nelluri) [inline pkg/cinder/deploy/kubernetes/csi-configmap-cinderplugin.yaml]: This has to be secret as it contains user name & password.

**Comment by chakri-nelluri** [comment]
Thanks @edisonxiang . Added minor comments.

File: pkg/cinder/README.md [hunk 4]
```diff
@@ -1,19 +1,58 @@
 # CSI Cinder driver
 
-## Usage:
+## Kubernetes
+
+### Requirements
+
+The folllowing feature gates and runtime config have to be enabled to deploy the driver.
```
Comment (dims) [inline pkg/cinder/README.md]: typo - `following`

File: pkg/cinder/README.md [hunk 5]
```diff
@@ -1,19 +1,58 @@
...
+
+```
+FEATURE_GATES=CSIPersistentVolume=true,MountPropagation=true
+RUNTIME_CONFIG="storage.k8s.io/v1alpha1=true"
+```
+
+Mountprogpation requries support for privileged containers. So, make sure privileged containers are enabled in the cluster.
```
Comment (dims) [inline pkg/cinder/README.md]: typo - `requires`

File: pkg/cinder/README.md (hunk 2 above)
Comment (dims) [reply pkg/cinder/README.md]: good point! agree

File: pkg/cinder/deploy/kubernetes/csi-configmap-cinderplugin.yaml (hunk 3 above)
Comment (dims) [reply pkg/cinder/deploy/kubernetes/csi-configmap-cinderplugin.yaml]: +1

**Comment by edisonxiang** [comment]
@chakri-nelluri @dims @j-griffith Thanks your suggestions. I will update asap.

File: pkg/cinder/README.md (hunk 1 above)
Comment (edisonxiang) [reply pkg/cinder/README.md]: Done

File: pkg/cinder/README.md (hunk 2 above)
Comment (edisonxiang) [reply pkg/cinder/README.md]: Done

File: pkg/cinder/deploy/kubernetes/csi-configmap-cinderplugin.yaml (hunk 3 above)
Comment (edisonxiang) [reply pkg/cinder/deploy/kubernetes/csi-configmap-cinderplugin.yaml]: Done

File: pkg/cinder/README.md (hunk 4 above)
Comment (edisonxiang) [reply pkg/cinder/README.md]: Done

File: pkg/cinder/README.md (hunk 5 above)
Comment (edisonxiang) [reply pkg/cinder/README.md]: Done

**Comment by edisonxiang** [comment]
@chakri-nelluri @dims @j-griffith Done. PTAL. Thanks.

File: pkg/cinder/README.md [hunk 6]
```diff
@@ -4,36 +4,29 @@
...
+Mountprogpation requires support for privileged containers. So, make sure privileged containers are enabled in the cluster.
 
 ### Example local-up-cluster.sh
 
 ```ALLOW_PRIVILEGED=true FEATURE_GATES=CSIPersistentVolume=true,MountPropagation=true RUNTIME_CONFIG="storage.k8s.io/v1alpha1=true" LOG_LEVEL=5 hack/local-up-cluster.sh```
 
 ### Deploy
 
-Please update ```cloud.conf``` informations in csi-configmap-cinderplugin.yaml file.
+Encode your ```cloud.conf``` file content by base64.
```
Comment (dims) [inline pkg/cinder/README.md]: "by" -> "using"

**Comment by dims** [comment]
tiny nit, looks good to me @edisonxiang

**Comment by edisonxiang** [comment]
@dims Done. Thanks.

**Comment by j-griffith** [comment]
LGTM unclear on how folks feel about squashing to a single commit per/pr; but that's a process and preference question and there's enough variation here that keeping each commit smaller is nice.

**Comment by chakri-nelluri** [comment]
@edisonxiang Looks good. Please squash it to single commit and will merge. Thanks

**Comment by edisonxiang** [comment]
@chakri-nelluri Done. Thanks.

**Comment by chakri-nelluri** [comment]
@edisonxiang Please rebase the merge commit too. Thanks.

**Pull request closed without merging**